jfather.json.txt

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Make sure you aren't committing your other `.json` files, refer to `sample.json` or add `jfather.json.txt` to your `.gitignore` file